kmod/src/inode.c

```diff
@@ -217,6 +217,19 @@ static u64 get_item_minor(struct scoutfs_inode_info *si, u8 type)
 	return si->item_minors[ind];
 }
 
+/* Returns the max inode size given format version */
```

Comment:
This doesn't belong in the worm patch, but if load_inode isn't calling it anymore then it won't need to be moved up here.